ColorPatterns/Model/GameTimer.swift

```diff
@@ -8,18 +8,20 @@
 
 import Foundation
 
-protocol GameTimerDelegate {
+protocol GameTimerDelegate: class {
     func timerEnded()
     func timerUpdate()
 }
 
 class GameTimer {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/134e78896b7d06b391bc723303fb96a8b747df75/ColorPatterns/Model/GameTimer.swift#L16"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>